internal/livelint/check_can_access_app.go

```diff
@@ -81,7 +81,7 @@ func (n *Livelint) canPortForward(pod apiv1.Pod, port int32, checkFunc func(uint
 	for range readyChan {
 	}
 
-	n.ui.Send(showSpinnerMsg{showing: true})
+	n.ui.StartSpinner()
 
 	// send some traffic via the port forwarding
 	forwardedPorts, _ := forwarder.GetPorts()
```

Comment:
note: not for now, but is ignoring the errors on port forwarding really the error handling we want here?